### ISSUE TITLE
bug(select): Down-arrow doesn't display in High Contrast mode

### ISSUE THREAD

**Issue by lukeschaefer** [angular/components#21263]
#### Reproduction

This bug can be replicated in the example documentation itself:
https://material.angular.io/components/select/examples

Steps to reproduce:
1. In Windows, turn on High-Contrast mode 
2. Open the examples in Firefox (chrome doesn't work with Windows High Contrast mode)
 
 
#### Expected Behavior

The down-arrow on the right side of the select box should be visible

#### Actual Behavior

Instead, it looks like a flat line

#### Environment

- Angular:
- CDK/Material:
- Browser(s): Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by crisbeto** [comment]
There's a PR for this already, but it wasn't marked as high priority so we never got around to merging it in. https://github.com/angular/components/pull/14219

**Comment by annieyw** [comment]
duplicate of https://github.com/angular/components/issues/14207
